### DISCUSSION TITLE
Ethernet switch

### DISCUSSION THREAD

**Discussion by wesleyscaldwell** [tiny-pilot/tinypilot#815] [Ideas]
First, thanks, this tool has saved me a lot of time standing at servers and user stations.

**Issue:**
One ethernet cable at remote pc - requires network switch/hub to connect both tinypilot and PC.

**Idea:**
Include/Sell an adapter that would be connected to the PI in the same way IP phones extend ethernet connections.

This might be a small or narrow requirement and might not need any attention.

**Comment by mtlynch** [comment]
Thanks for the suggestion! We haven't gotten a lot of requests for this, but I'll keep it in mind if other users hit the same problem.

In the meantime, I'd recommend an inexpensive unmanaged switch. I use the [TP-Link TL-SG105](https://www.newegg.com/tp-link-tl-sg105-5-x-rj45/p/N82E16833704179?item=N82E16833704179) ($13), and it's performed well for me for several years.

**Reply by Stubblemonster** [reply]
I am building this at the moment. I am using a dual USB charger with 4.1amp capacity, one USB will go to the TinyPilot Power and the other to a UniFi Flex Mini switch. The great thing about the UniFi is it's very cheap, small, managed via the cloud (I can see the IP of the Pi from the interface) and very low power. I'm going to encompass in a 3d printed case, only exposing the free ports so the end user only needs to plug in the PC lan cable into it and a supplied short lan cable to the PC. Then they just have USB to PC and one power lead.
